requirements.txt

```diff
@@ -1,6 +1,7 @@
 codecov==2.0.15
 coverage==4.5.2
 cloudpickle==1.2.1  # needed for dask
+cvxpy==1.0.25
```

Comment:
Can we use scipy methods instead? (too many dependencies are not good, that is why there should serious reason for dependency to be added )

Reply:
@proskurin The reason I did not use scipy is because cvxopt's code is easier to understand by others and is very simple. Scipy's implementation becomes a little complex to understand

Reply:
Having a look at other packages, I think making use of cvxopt may be unavoidable. It comes down to a design choice. Whilst I don't like having more package dependencies added, I understand if you want to opt for this over scipy.

Reply:
I think it’s ok having packages well know by comunnity like kaggle packages or anaconda python packages

Cvxopt is wellknow to execute second order optimizations (markovitz)

Reply:
Example

https://cvxopt.org/examples/book/portfolio.html